composer.json

```diff
@@ -89,10 +88,6 @@
       }
     },
     "patches": {
-      "phpunit/phpunit-mock-objects": {
-        "Fix ReflectionParameter warnings on PHP 8": "patches/phpunit-mock-objects.patch",
-        "Remove 'match' keyword from uses <https://github.com/sebastianbergmann/phpunit/issues/4373>": "patches/remove-match-keyword.patch"
```

Comment:
These files can be deleted from the repo now, correct?

Reply:
Yes and I have removed them as well.